libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Queue/CSFNetwork.m

```diff
@@ -109,17 +110,30 @@ + (instancetype)networkForUserAccount:(SFUserAccount*)account {
     return instance;
 }
 
++ (instancetype)networkForUnauthenticatedUser {
+    CSFNetwork *instance = nil;
+    
+    @synchronized (SharedInstances) {
+        instance = SharedInstances[CSFUnauthenticatedUserKey];
+        if (!instance) {
+            instance = SharedInstances[CSFUnauthenticatedUserKey] = [[self alloc] init];
+        }
+    }
+    
+    return instance;
+}
+
 - (id)init {
     self = [super init];
     if (self) {
         self.queue = [NSOperationQueue new];
         [self.queue addObserver:self forKeyPath:@"operationCount" options:NSKeyValueObservingOptionNew context:kObservingKey];
         _online = YES;
-        
-        // Start the queue suspended, so we can unsuspend it when the user account object is set
-        self.queue.suspended = YES;
-        _networkSuspended = YES;
-        
+
+        // Start the queue unsuspended, since the queue could be used by an unauthenticated user.
+        self.queue.suspended = NO;
```

Comment:
The default value of -[NSOperationQueue suspended] is already NO, so you don't have to do that here.